include/entity.h

```diff
@@ -1852,7 +1852,7 @@ typedef struct {
     /* 0x7C */ s32 : 32;
     /* 0x80 */ u32 resetTimer;
     /* 0x84 */ u32 breakCount;
-} ET_BreakableCrystalFloor;
+} ET_BreakableTerrain;
```

Comment:
The same ext structure is used in NO4 for both the crystal floor and the breakable wall. I didn't find an equivalent that matches other overlays, and it didn't make much sense to separate them into separate structs IMO, so I have combined the two and renamed slightly to reflect.

Happy to update this if something else is preferred naming wise, or splitting into separate structs.